<ui version="4.0" >
 <class>BIGHOME_UI_MainWindow_titleBarClass</class>
 <widget class="QWidget" name="BIGHOME_UI_MainWindow_titleBarClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>23</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>15</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>23</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>BIGHOME_UI_MainWindow_titleBar</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>309</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="smallBtn" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>14</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>14</height>
        </size>
       </property>
       <property name="text" >
        <string>smallBtn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>14</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>14</height>
        </size>
       </property>
       <property name="text" >
        <string>closeBtn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutBtn" >
       <property name="minimumSize" >
        <size>
         <width>26</width>
         <height>14</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>26</width>
         <height>14</height>
        </size>
       </property>
       <property name="text" >
        <string>aboutBtn</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
